corehq/apps/linked_domain/templates/linked_domain/domain_links.html

```diff
@@ -23,144 +23,156 @@
       {% endif %}
-      <li{% if not is_linked_domain %} class="active"{% endif %}><a data-toggle="tab" href="#tabs-linked">{% trans "Projects linked to this one" %}</a></li>
-      <li><a data-toggle="tab" href="#tabs-release">{% trans "Release Content" %}</a></li>
+      <li{% if not is_linked_domain %} class="active"{% endif %}><a data-toggle="tab" href="#tabs-manage-downstream">{% trans "Projects linked to this one" %}</a></li>
+      <li><a data-toggle="tab" href="#tabs-push-content">{% trans "Release Content" %}</a></li>
     </ul>
     <div class="spacer"></div>
   {% elif is_linked_domain %}
     <h2>{% trans "Project Link" %}</h2>  {# header for the only tab that will be displayed #}
   {% endif %}
 
-  <div id="domain_links" class="ko-template">
+  <div id="domain_links">
     <div class="tab-content">
       {% if is_linked_domain %}
-        <div class="tab-pane fade in active" id="tabs-master">
-          <div data-bind="if: master_link">
-            <p>{% trans "This project is linked to " %}<a data-bind="attr: {'href': master_href}, text: master_link.master_domain"></a></p>
-            <div>
-              <table class="table table-striped table-hover">
-                <thead>
-                <tr>
-                  <th class="col-sm-5">{% trans "Linked Model" %}</th>
-                  <th class="col-sm-2">{% trans "Last Updated" %} ({{ timezone }})</th>
-                  <th class="col-sm-5"></th>
-                </tr>
-                </thead>
-                <tbody data-bind="foreach: model_status">
-                <tr>
-                  <td data-bind="text: name"></td>
-                  <td data-bind="text: last_update"></td>
-                  <td data-bind="css: {'has-error': error}">
-                    <button class="btn btn-danger" data-bind="visible: showUpdate() && !update_url, click: update">
-                      {% trans "Overwrite" %}
-                    </button>
-                    <button class="btn btn-default disabled" data-bind="visible: showSpinner">
-                      <i class="fa fa-spinner"></i>
-                    </button>
-                    <button class="btn btn-success disabled" data-bind="visible: hasSuccess">
-                      <i class="fa fa-check"></i> {% trans "Success" %}
-                    </button>
-                    <button class="btn btn-danger disabled" data-bind="visible: error">
-                      <i class="fa fa-times"></i> {% trans "Error" %}
-                    </button>
-                    <span class="help-block" data-bind="visible: error, text: error"></span>
-                    <a data-bind="visible: showUpdate && update_url, attr: {href: update_url}">{% trans "Go to update page" %}</a>
-                  </td>
-                </tr>
-                </tbody>
-              </table>
+        <div class="tab-pane fade in active" id="tabs-pull-content">
+          <div id="ko-tabs-pull-content" class="ko-template">
```

Comment:
Nitpick: what's the extra layer for?

Reply:
@biyeun might be able to speak better to this than I can, but I'll give it a shot. The intention is to separate the knockout element from the tab/bootstrap element. At one point I did have an issue with the tab not rendering appropriately and this change appeared to that fix that, but I'm not positive the same applies for the code in its current state.

Reply:
Interesting. I can see the logic of separating the styling and the interactivity.